docs/apm/service-maps.asciidoc

```diff
@@ -0,0 +1,50 @@
+[[service-maps]]
+=== Service maps
+
+beta::[]
+
+A service map is a real-time diagram of the interactions occurring in your application’s architecture.
+It allows you to easily visualize data flow, and high-level statistics, like average transaction duration,
+requests per minute, errors per minute, and metrics, allow you to quickly assess the status of services.
+
+Our beta offering creates two types of service maps:
+
+* Global: All services and connections are shown.
+* Localized: Selecting a specific service will highlight it's connections.
```

Comment:
I don't know if we actually have names for these but I don't know if "Localized" is the right word. I would call it "Service-specific."